ui/StatusQ/include/StatusQ/keychain.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+#include <QObject>
+
+class Keychain : public QObject {
+    Q_OBJECT
+
+    Q_PROPERTY(QString service READ service WRITE setService NOTIFY serviceChanged)
+    Q_PROPERTY(QString reason READ reason WRITE setReason NOTIFY reasonChanged)
+    Q_PROPERTY(bool loading READ loading NOTIFY loadingChanged)
+
+public:
+    explicit Keychain(QObject *parent = nullptr);
+    ~Keychain();
```

Comment:
No need to define (or even declare) an empty DTOR

Reply:
It's not empty anymore 👌